packages/commonwealth/client/scripts/state/api/topics/fetchTopics.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import app from 'state';
+import { ApiEndpoints } from 'state/api/config';
+import Topic from 'models/Topic';
+import { useQuery } from '@tanstack/react-query';
+
+const TOPICS_STALE_TIME = 30 * 1_000; // 30 s
```

Comment:
there is no golden mean for stale time. Each query can have its own. To be able to modify this value we should know 100% how staleTime vs cacheTime work and what are the implications. Great reading here https://tkdodo.eu/blog/react-query-as-a-state-manager#takeaways

Generally we should start with default stale and then try to modify it for our needs